Sources/Core/KeystoreManager/BIP32Keystore.swift

```diff
@@ -98,6 +97,18 @@ public class BIP32Keystore: AbstractKeystore {
         try self.init(seed: seed, password: password, prefixPath: prefixPath, aesMode: aesMode)
     }
 
+    //TODO: Unit Test
+    //TODO: merge and cleanup with above code
+    public convenience init?(mnemonicsPhrase: [String], password: String, mnemonicsPassword: String = "", language: BIP39Language = .english, prefixPath: String = HDNode.defaultPathMetamaskPrefix, aesMode: String = "aes-128-cbc") throws {
```

Comment:
Suggestion: add newline characters after each comma so that it's easier to read.